soccer/gameplay/tactics/positions/submissive_defender.py

```diff
@@ -29,10 +34,45 @@ def __init__(self):
 
         self.add_state(SubmissiveDefender.State.marking,
                        behavior.Behavior.State.running)
+        self.add_state(SubmissiveDefender.State.clearing,
+                       behavior.Behavior.State.running)
 
         self.add_transition(behavior.Behavior.State.start,
                             SubmissiveDefender.State.marking, lambda: True,
                             "immediately")
+        self.add_transition(SubmissiveDefender.State.marking,
+                            SubmissiveDefender.State.clearing,
+                            lambda: self.go_clear,
+                            "when it is safe to clear the ball")
+        self.add_transition(
+            SubmissiveDefender.State.clearing,
+            SubmissiveDefender.State.marking,
+            lambda: self.subbehavior_with_name('kick-clear').state == behavior.Behavior.State.completed or not self.should_clear_ball(evaluation.ball.time_to_ball(self.robot)),
+            "done clearing")
+
+    def should_clear_ball(self, our_time_to_ball):
+        #Returns true if our robot can reach the ball sooner than the closest opponent
+        if main.ball().pos.mag() < constants.Field.ArcRadius * 2:
```

Comment:
@justbuchanan If I remove that call then it would still work but the rest of the if statements after it would get called a lot, if that's ok then I can remove it. It's necessary in the other spot because the robot needs to be able to escape the clearing behavior if the ball rolls out of the radius.